tests/invalid/table/injection-1.toml

```diff
@@ -0,0 +1,6 @@
+[a.b.c]
+  z = 9
+[a]
+  b.c.t = "injecting into [a.b.c] defined above is not allowed"
```

Comment:
Don't want to be overly pedantic about verbiage, but I find the usage of "injecting" rather confusing. I suppose it's technically correct, but how I use it is as security vulnerability rather than to mean "override". Wouldn't replacing "injection into" with "overriding" make things clearer?

Reply:
I don't think "overriding" is correct, though, since it (to me, at least) implies that the initial definition is being replaced altogether. How about something based around either "appending" or "modifying"?

Reply:
Yeah, sure; that's fine too. It's just that with "injecting" I think about some sort of security issue, but maybe that's just me 😅 Thanks!